autoload/keepeye/system/bell.vim

```diff
@@ -1,6 +1,7 @@
 function! keepeye#system#bell#Trigger(message)
-  if executable('mpv')
-    call system('mpv ' . shellescape(g:keepeye_bell_path) . '&')
+  let l:player = substitute(g:keepeye_bell_cmd, '\s', '', 'g')
+  if executable(l:player)
+    call system(l:player . ' ' . shellescape(g:keepeye_bell_cmd_flags) . ' ' . shellescape(g:keepeye_bell_path) . '&')
```

Comment:
I would use the `join()` function to have a nicer render here, sth like `let l:cmd = join([l:player, shellescape(g:keepeye_bell_cmd_flags), shellescape(g:keepeye_bell_path), '&'], ' ')`